core.d.ts

```diff
@@ -476,6 +476,9 @@ export declare class TokenizerPositionError extends Error {
 export declare class FileTypeParser {
 	detectors: Iterable<Detector>;
 
+	/**
+	 @param options -  `options.customDetectors`: list of custom detectors
+	 */
```

Comment:
Doc comments are not needed if they don't add anything to the types. Here, the doc comments just repeats the information available in the below types.